BoardWebApp-Tests/LoginValidationTests.cs

```diff
@@ -1,38 +1,39 @@
-﻿using BoardWebApp.Controllers;
-using BoardWebApp.Models;
-using BoardWebApp.ViewModels;
-using Microsoft.AspNetCore.Mvc;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using Xunit;
 using Xunit.Abstractions;
+using BoardWebApp.Controllers;
+using BoardWebApp.Models;
+using BoardWebApp.ViewModels;
+using Microsoft.AspNetCore.Mvc;
 
 namespace BoardWebApp_Tests
 {
-    public class LoginValidationTests
+    public class LoginValidationTests : IDisposable
     {
         private BoardWebAppContext _dbContext;
         private readonly ITestOutputHelper _output;
 
         public LoginValidationTests(ITestOutputHelper testOutputHelper)
         {
             _output = testOutputHelper;
+            _dbContext = InMemoryDb.InitInMemoryDbContext();
+        }
+        public void Dispose()
+        {
+            _dbContext.Database.EnsureDeleted();;
         }
 
+
+        // --------------------- This test verifies that whenever a user fails to login, they're displayed with the same page and an error message is to the view. ---------------------
         [Theory]
-        //[InlineData("trudy@board.com", "P@ssword1", true)] // All good
+        //[InlineData("trudy@board.com", "P@ssword1", true)] // All good --- Pavel: I can't mock the Http request. Give it a go if you want, would be good test to make sure we don't break it.
```

Comment:
What's this Pav, with regards to mocking the http request ?

Reply:
I remember saying this before at some point. That test does not return true -> It throws a NullReferenceException as soon as the execution gets to the line on AccountController that add's a cookie to the response.
Since this test project does not run a server there's no HTTP request - response. There are ways of somehow mocking those but I haven't understood them just yet.